### ISSUE TITLE
Feature request: hardware control like pulse

### ISSUE THREAD

**Issue by Memcys** [ScQ-Cloud/pyquafu#12]
I wonder if it is possible to implement a programming kit for hardware-level control  like [`qiskit.pulse`](https://qiskit.org/documentation/apidoc/pulse.html#pulse-qiskit-pulse), see
- [skit pulse: Pro-gramming quantum computers through the cloud with pulses,Quantum Sci. Technol.5, 044006 (2020)](https://iopscience.iop.org/article/10.1088/2058-9565/aba404)

**Comment by chensgit169** [comment]
Yes, we have been working on to support pulse instructions. Please keep on to see the update and leave any of your concrete suggestions for development of this function.
